CONTRIBUTING.md

```diff
@@ -217,3 +216,21 @@ npm run release -- --beta
 ```
 
 _Make sure to use `npm run` instead of `yarn run` to avoid issues._
+
+#### Experimental TypeScript version
+
+An experimental version containing the TypeScript declaration files is available.
+
+Since some of these declaration files are generated from the JSDoc comments they can still contains some typing errors. This version will stay experimental until we are confident enough in the generated declarations to put them in the stable release.
```

Comment:
```suggestion
Since some of these declaration files are generated from the JSDoc comments, they can contain some typing errors. This version will stay experimental until we are confident enough in the generated declarations to put them in a stable release.
```